src/views/BottomTabBar.js

```diff
@@ -240,6 +251,9 @@ class TabBarBottom extends React.Component<Props> {
               onLongPress={() => onTabLongPress({ route })}
               testID={testID}
               accessibilityLabel={accessibilityLabel}
+              accessibilityRole={accessibilityRole}
+              accessibilityStates={accessibilityStates}
+              focused={focused}
```

Comment:
I also exposed this to the `ButtonComponent` to handle the case of a custom component. When using a custom component, I was not able to find a way to determine whether or not that it was focused, so I could not determine `accessibilityStates`.

Reply:
Let's keep this PR focused on accessibility. I think there is some way to check if it's focused and I'm not sure about the API. So I'd have to spend more time to merge it if it's going to be in the same PR

Reply:
I removed the focused property for this PR. Though, it's still unclear how a custom component would become aware of whether or not it's focused without it. However, I think with the other changes, I would no longer need to create a custom component and could just use the default.